contracts/domainRegister.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.24;
+
+contract DomainRegister {
+    struct DomainInfo {
+        address controller;
+        uint256 registrationTime;
+    }
+    uint256 public fee;
+    address public owner;
+    uint public totalDomains;
+    mapping(string => DomainInfo) public domains;
+    mapping(address => string[]) public controllerDomains;
+
+    event DomainRegistered(
```

Comment:
line 15
event contains time stamp, same as block.timestamp when event is emitted
// - 1 point